server/grpc/server.go

```diff
@@ -107,12 +103,12 @@ func (s *Server) Close() {
 // register all server
 func (s *Server) register() error {
 	coreServer := core.NewServer(s.sdk)
-	newCoreServer := newcore.NewServer(s.serviceSDK)
+	coreServiceServer := NewServiceServer(s.sdk)
```

Comment:
could pass only the `s.sdk.Service` to reduce dependencies but it's fine ;)

Reply:
👍 on forum proposal it says that sub sdk packages can use each other so this why chose to use sdk